main.go

```diff
@@ -104,19 +107,9 @@ func main() {
 		HealthHandler:        handlers.MakeHealthHandler(),
 		InfoHandler:          handlers.MakeInfoHandler(version.BuildVersion(), version.GitCommit),
 		SecretHandler:        handlers.MakeSecretHandler(functionNamespace, clientset),
-		LogHandler:           logs.NewLogHandlerFunc(handlers.NewLogRequestor(clientset, functionNamespace), cfg.WriteTimeout),
+		LogHandler:           logs.NewLogHandlerFunc(handlers.NewLogRequestor(clientset, functionNamespace), cfg.FaaSConfig.WriteTimeout),
```

Comment:
I don't want to block the PR, but I am not sure with this naming: `BootStrapConfig.FaaSConfig`, could it be something more descriptive for the `FaaSConfig`?